CHANGELOG.md

```diff
@@ -1,3 +1,5 @@
+## 8.5.0
+- Added support for new payment initiation options
```

Comment:
We typically add to the changelog when releasing rather than when making changes